### PR TITLE
{vis}[intel/2017a] FFmpeg v3.3.4, x264 v20170913

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2470.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/906337cc0d530e90b1db414af8bd125b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2109.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5eeb1ad630dd23de44cbf6f5425cd336 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2425.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7687437a9f1aed240569efdf471151ba for a full test report.

File: easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.3.4-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+source_urls = ['http://ffmpeg.org/releases/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['5ef5e9276c311c74ab2e9d301c2d7ee10e1f2cbd758c6f13d6cb9514dffbac7e']
+
+dependencies = [
+    ('NASM', '2.12.02'),
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.6'),
+    ('x264', '20170913'),
```
Comment (verdurin) [inline easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.3.4-intel-2017a.eb]: @boegel any reason why `LAME` and `x265` were removed from the dependencies? Upstream changes?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in this PR)
ca043.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/615d249285658796d21efab54038fbb3 for a full test report.

File: easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.3.4-intel-2017a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.3.4-intel-2017a.eb]: Hmm, that wasn't the intention... Will fix.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2011.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/712134d4da2e2f19ea1230128eea4dc4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (4 easyconfigs in this PR)
node2700.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/26af956337b7a6c9ad7daa432260341a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (4 easyconfigs in this PR)
node2700.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7e2d3769d14c8dc8fec1339c2914397e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2011.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cd1dbba052de117bdb411480dee897ea for a full test report.

**Comment by boegel** [comment]
@verdurin Good to go now?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
ca001.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4b4905208148ded27a5a33acd5d26478 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**